integration/integration-Full.spec.ts

```diff
@@ -133,4 +135,24 @@ test("Upload configuration file, choose form, fill form, preview form, submit fo
   await t.expect(Title.textContent).contains("Document(s) issued successfully");
   await t.expect(Selector("div").withText("Document-1.tt").exists).ok();
   await t.expect(Selector("a[download='Document-1.tt']").exists).ok();
+
+  // Failed publish document
+  await t.click(LogoutButton);
+  await t.click(ButtonReset);
+  await t.setFilesToUpload("input[type=file]", [ConfigFailPublishDocument]);
+  await t.typeText(PasswordField, "password");
+  await t.click(ButtonLogin);
+  await t.expect(Title.textContent).contains("Choose Document Type to Issue");
+  await t.click(Button.withText("Covering Letter (GT)"));
+  await t.click(SubmitButton);
+  await t.expect(Title.textContent).contains("Document(s) failed to issue");
+  await t.expect(Selector("div").withText("1 Document(s) Failed").exists).ok();
+  await t
+    .expect(
+      Selector("div").withText(
+        "These documents failed to publish due to some errors. Kindly rectify and try publishing again."
+      ).exists
+    )
+    .ok();
+  await t.expect(Selector("div").withText("Document-1.tt").exists).ok();
```

Comment:
Can we start breaking down integration test to individual test files? The test itself is becoming really hard to read.

Reply:
ok sure